geopandas/base.py

```diff
@@ -189,6 +189,9 @@ def unary_geo(self, op):
         x = vectorized.geo_unary_op(op, self._geometry_array.data)
         return GeoSeries(x, index=self.index, crs=self.crs)
 
+    def coords(self):
+        return vectorized.get_coordinates(self._geometry_array.data)
```

Comment:
Not sure this is needed as a public method (would need to think about usecases). Eg for points you already have x/y as attributes to get the coordinates.